java/client/src/org/openqa/selenium/remote/tracing/EventAttributeValue.java

```diff
@@ -64,7 +96,11 @@ public Type getAttributeType() {
     STRING,
     BOOLEAN,
     LONG,
-    DOUBLE
+    DOUBLE,
```

Comment:
Since we've not shipped this, please alphabetise. Once we've shipped, it's best to maintain the ordinals of the enum, but for now it doesn't matter.